src/napari/layers/labels/labels.py

```diff
@@ -1573,15 +1573,27 @@ def _get_tooltip_text(
         msg : string
             String containing a message that can be used as a tooltip.
         """
-        return '\n'.join(
-            self._get_properties(
-                position,
-                view_direction=view_direction,
-                dims_displayed=dims_displayed,
-                world=world,
-            )
+        value = self.get_value(
+            position,
+            view_direction=view_direction,
+            dims_displayed=dims_displayed,
+            world=world,
+        )
+        if value is None:
+            return ''
+
+        properties = self._get_properties(
+            position,
+            view_direction=view_direction,
+            dims_displayed=dims_displayed,
+            world=world,
         )
 
+        if not properties:
+            return f'{value}'
+
+        return f'{value}\n' + '\n'.join(properties)
```

Comment:
Because value is otherwise not identified (like with a dict key), perhaps we do:

```suggestion
        return f'value: {value}\n' + '\n'.join(properties)
```

and propagate to other pathways as well

Reply:
I had that, but then opted against to match the status bar...

Reply:
Fair enough, I think with my lil screenshot showing the tooltip text it looks much more out of place, especially because I can imagine that the status bar is giving a value at a position, but the hover tooltip is just a number? Hope others can also give feedback

Reply:
I could go either way. I might suggest "label: " instead of "value".